src/registrar/models/domain_application.py

```diff
@@ -541,6 +570,14 @@ def approve(self):
             user=self.creator, domain=created_domain, role=UserDomainRole.Roles.ADMIN
         )
 
+    @transition(field="status", source=SUBMITTED, target=INVESTIGATING)
+    def in_review(self, updated_domain_application):
```

Comment:
**suggestion**: This transition method is different than the other ones because it uses an additional argument and then uses that object to send the email message. Could you explain more in this method's docstring about why it is that way?

Reply:
Yep! I commented to this in admin.py's `save_model` and was wondering whether I should document it on this side as well.